Package.swift

```diff
@@ -21,22 +23,51 @@ let package = Package(
         .target(
             name: "ClientRuntime",
             dependencies: [
+                "SmithyXML",
                 .product(name: "AwsCommonRuntimeKit", package: "aws-crt-swift"),
                 .product(name: "Logging", package: "swift-log"),
                 .product(name: "XMLCoder", package: "XMLCoder")
             ]
         ),
-        .testTarget(
-            name: "ClientRuntimeTests",
-            dependencies: ["ClientRuntime", "SmithyTestUtil"]
+        .target(name: "SmithyReadWrite"),
+        .target(
+            name: "SmithyXML",
+            dependencies: [
+                "SmithyReadWrite",
+                "SmithyTimestamps",
+                .target(name: "LibXML2", condition: .when(platforms: [.linux]))
```

Comment:
Is it possible to have the same name as the macOS counterpart so that we don't have to use #if else in the code? I don't know, but it's worth checking out.

Reply:
Done.  For some reason I was thinking it was not permitted, but it worked fine.